server/tests/test_route_newsletter.py

```
@@ -0,0 +1,147 @@
+import json
+from models.newsletter_subscriber import NewsletterSubscriber
+
+
+def test_subscribe_newsletter_successfully(client, session):
```

Ruff (0.13.1)

Unused function argument: `session`

(ARG001)